### DISCUSSION TITLE
Suggestion: Zombrex Airdrops

### DISCUSSION THREAD

**Discussion by MalcolmMorin1998** [TEHE-Studios/ExpandedHelicopterEvents#323] [Q&A]
As the title suggests, this would make EHE compatible with Zombrex in the sense that it adds a special helicopter event that ONLY works if Zombrex is enabled alongside it.

This special helicopter event behaves just like FEMA and Samaritan airdrops, except it only drops supplies of Zombrex. By default, a single dose of Zombrex lasts 72 in-game hours, so the default amount of Zombrex dropped would keep an infected player alive for 30 in-game days, so 10 doses of Zombrex. This amount could be changed in the sandbox options in order to account for different Zombrex settings; I'd say a hard limit for how many can be dropped per drop would be 100 doses.

By default, this would disable natural spawns (bathroom, pharmacy, etc.) for Zombrex in the game world, but this can also be disabled if the player chooses.

This setting could be useful for those trying to replicate Dying Light and want to attempt such a run.

**Comment by Chuckleberry-Finn** [comment]
I may add a sandbox in order to include an additional present or loot changes. It would need to be planned out well. Thanks for the suggestion.
